vllm/model_executor/model_loader/loader.py

```diff
@@ -786,6 +787,7 @@ def __init__(self, load_config: LoadConfig):
         with open(config_file_path, "r") as f:
             config = json.load(f)
             self.target_modules = config["target_modules"]
+        self.wo_sharded_weights_modules: List[str] = []
```

Comment:
What does `wo` stand for?

Reply:
wo is the abbreviation for `without`

Reply:
How about using `self.unsharded_weights_modules`? Because `wo` is also used as abbreviation for some attention's `out_proj` like molmo and original pixtral. Using `wo` might be misleading in some cases.

Reply:
Pretty reasonable.  I have made the changes.